spanner/tests/system/test_system.py

```diff
@@ -323,6 +323,24 @@ def _unit_of_work(transaction, test):
         rows = list(self._db.execute_sql(self.SQL))
         self._check_row_data(rows)
 
+    def test_db_run_in_transaction_twice(self):
+        # See issue #3434.
```

Comment:
I am :-1: on this comment in the source (a `git blame` accomplishes similar things)